packages/google_sign_in/google_sign_in/lib/google_sign_in.dart

```diff
@@ -286,20 +293,36 @@ class GoogleSignIn {
     return _currentUser;
   }
 
-  Future<void> _ensureInitialized() {
-    return _initialization ??=
-        GoogleSignInPlatform.instance.initWithParams(SignInInitParameters(
+  // Future that completes when we've finished calling `init` on the native side.
```

Comment:
Nit: "when `init` has completed on the native side."

Reply:
Yep, much better.